core/templates/dev/head/components/question-directives/questions-list/questions-list.directive.ts

```diff
@@ -140,7 +140,7 @@ angular.module('oppia').directive('questionsList', [
             }
           };
 
-          ctrl.saveAndPublishQuestion = function() {
+          ctrl.saveAndPublishQuestion = function(commitMessage = 'blank') {
```

Comment:
Why do you have the 'blank' here? I think that's not right -- if anything, you should have a check at the start of this function that verifies that a commit message is provided and throws an error if not.

Reply:
done